test/local/account_routes.js

```diff
@@ -121,3 +122,34 @@ test(
     })
   }
 )
+
+test(
+  '/recovery_email/status logs query reason',
+  function (t) {
+    var mockLog = {
+      begin: function () {},
+      increment: function (name) {
+        if (name === 'recovery_email_reason.push') {
+          t.end()
```

Comment:
Just to be sure I understand - if the route handler is not behaving as expected, the `increment` method will not be called, and the test will eventually time out with an error?  It might be a little more reliable to use the `reply` callback to detect when the handler has completed, and explicitly assert that the right function(s) were called on the mock.